std_execution.bs

```diff
@@ -1098,6 +1098,24 @@ execution::sender auto snd = execution::then(vals, [](auto... args) {
 
 This adaptor is included as it greatly simplifies lifting values into senders.
 
+### `execution::just_error` ### {#design-sender-factory-just_error}
+
+<pre highlight="c++">
+execution::sender auto just_error(
+    auto && error
+);
+</pre>
+
+Returns a sender with no [=completion scheduler|completion schedulers=], which completes with the specified error. If the provided error is an lvalue reference, a copy is made inside the returned sender and a non-const lvalue reference to the copy is sent to the receiver's `set_error`. If the provided value is an rvalue reference, it is moved into the returned sender and an rvalue reference to it is sent to the receiver's `set_error`.
+
+### `execution::just_done` ### {#design-sender-factory-just_done}
+
+<pre highlight="c++">
+execution::sender auto just_done();
```

Comment:
Do we want to define this as a type named `just_done` or a function named `just_done()`?

Reply:
I _think_ I prefer it as a function, but I could go either way, really.